src/chartinger/Behat/TwigReportExtension/views/default.twig

```diff
@@ -97,7 +97,15 @@
                   {% for step in feature.background.steps %}
                      <li class="list-group-item {% if step.result == 20 %}list-group-item-warning{% elseif step.result == 10%}disabled{% elseif step.result == 99 %}list-group-item-danger{% else %}list-group-item-success{% endif %}">
                      {{ default.result_glyph(step.result) }}
-                     {{ step.text }}
+                     {{ step.text }} 
```

Comment:
I haven't work before with twig templates. I don't know if there is more neat way to do this.